config/jobs/image-pushing/k8s-staging-test-infra.yaml

```diff
@@ -273,3 +273,30 @@ postsubmits:
           - --project=k8s-staging-test-infra
           - --env-passthrough=PULL_BASE_REF
           - triage/
+          
+periodics:
+  - name: periodic-test-infra-push-bootstrap
+    cluster: k8s-infra-prow-build-trusted
+    cron: "0 0 2 * *"
```

Comment:
+1 for infrequently building an updated image even if we don't need to change the dockerfile